### PR TITLE
Ensure column type is always string for CSV tables

### PR DESCRIPTION
**Pull request by wendigo**
OpenCSVSerde deserializes columns from csv file into strings only. While reading CSV-backed table we rely on fact that metastore will return column type being string, not a real schema-declared type. This patch ensures that for CSV table we would operate on string columns only.

### CODE REVIEW

**Comment by findepi** [comment]
https://github.com/prestosql/presto/blob/4df88adec3e0ea75c3127c80ff9f35678d62a4df/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java#L89 would need to be updated.

File: presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java [hunk 1]
```diff
@@ -86,9 +87,12 @@ public BridgingHiveMetastore(ThriftMetastore delegate)
     public Optional<Table> getTable(HiveIdentity identity, String databaseName, String tableName)
     {
         return delegate.getTable(identity, databaseName, tableName).map(table -> {
-            if (isAvroTableWithSchemaSet(table) || isCsvTable(table)) {
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java:89]: Also update https://prestosql.io/docs/current/connector/hive.html#metastore-configuration-for-avro-and-csv

it's no longer required for CSV.

**Comment by wendigo** [comment]
AC @findepi

File: presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java (hunk 1 above)
Comment (wendigo) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java:89]: 👍

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -55,10 +55,10 @@ The following file types are supported for the Hive connector:
 * CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
 * TextFile
 
-Metastore Configuration for Avro and CSV
+Metastore Configuration for Avro
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/hive.rst:58]: shorten the `^^^^` below

**Comment by findepi** [comment]
CI failed -- tz

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**